usaspending_api/recipient/v2/views/states.py

```diff
@@ -53,22 +53,36 @@ def obtain_state_totals(fips, year=None, award_type_codes=None, subawards=False)
         queryset = (
             matview_search_filter(filters, SummaryStateView)
             .values("pop_state_code")
-            .annotate(total=Sum("generated_pragmatic_obligation"), distinct_awards=StringAgg("distinct_awards", ","))
-            .values("distinct_awards", "pop_state_code", "total")
+            .annotate(
+                total=Sum("generated_pragmatic_obligation"),
+                distinct_awards=StringAgg("distinct_awards", ","),
+                total_face_value_loan_amount=Sum("face_value_loan_guarantee"),
+            )
+            .values("distinct_awards", "pop_state_code", "total", "total_face_value_loan_amount")
         )
 
+        total_face_value_loan_prime_awards = queryset.filter(type__in=["07", "08"]).count()
```

Comment:
When testing locally, this value was always `1`, likely due to having this filter come after the annotation/ summing of the total loan amount.